src/apps/fuzzers/fuzzerPolygonToCells.c

```diff
@@ -14,11 +14,12 @@
  * limitations under the License.
  */
 /** @file
- * @brief Fuzzer program for polygonToCells and related functions
+ * @brief Fuzzer program for polygonToCellsExperimental and related functions
```

Comment:
This is incorrect, right? This fuzzes the old function

Reply:
Yes, that's a mistake